src/e2e-test/features/gcs/sink/GCSSink.feature

```diff
@@ -199,3 +199,73 @@ Feature: GCS sink - Verification of GCS Sink plugin
+    Then Read Credentials file "serviceAccountJSON" with value: "keys"
+    Then Enter the BigQuery source mandatory properties
+  #  Then Enter input plugin property: "table" with value: "bqTableName"
+    Then Click on the Get Schema button
+#    Then Change datatype of fields in output schema with : "bqDatatypeChange1"
+    Then Validate "BigQuery" plugin properties
+#    Then Verify the Output Schema matches the Expected Schema: "bqDataTypeTestFileSchema1"
+    Then Verify the Output Schema matches the Expected Schema: "bqSourceSchemaDatatype"
+    Then Close the BigQuery properties
+    Then Open GCS sink properties
+    Then Enter GCS property projectId and reference name
+    Then Select radio button plugin property: "serviceAccountType" with value: "JSON"
+    Then Read Credentials file "serviceAccountJSON" with value: "keys"
+    Then Enter GCS sink property path
+#    Then Enter input plugin property: "path" with value: "gcsTargetPath"
+    Then Select GCS property format "json"
+#    Then Enter GCS sink property path suffix "gcsPathSuffix"
+    Then Validate "GCS" plugin properties
+    Then Close the GCS properties
+    Then Save and Deploy Pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Verify data is transferred to target GCS bucket
+    Then Close the pipeline logs
+    Then Validate output file generated by GCS sink plugin is equal to expected output file "BQToDBAllDataTypes"
+#    Then Validate the values of records transferred to target GCS is equal to the values from source BigQuery table
+
+  @BQ_SOURCE_DATATYPE_TEST @GCS_SINK_TEST
+  Scenario: To verify data is getting transferred successfully from BigQuery to GCS using delimiter format
+    Given Open Datafusion Project to configure pipeline
+    When Source is BigQuery
+    When Sink is GCS
+    Then Connect source as "BigQuery" and sink as "GCS" to establish connection
+    Then Open BigQuery source properties
+    Then Select radio button plugin property: "serviceAccountType" with value: "JSON"
+    Then Read Credentials file "serviceAccountJSON" with value: "keys"
+    Then Enter the BigQuery source mandatory properties
+#    Then Enter input plugin property: "table" with value: "bqTableName"
+    Then Click on the Get Schema button
+    Then Validate "BigQuery" plugin properties
+    Then Verify the Output Schema matches the Expected Schema: "bqSourceSchemaDatatype"
+    Then Close the BigQuery properties
+    Then Open GCS sink properties
+    Then Enter GCS property projectId and reference name
+    Then Select radio button plugin property: "serviceAccountType" with value: "JSON"
+    Then Read Credentials file "serviceAccountJSON" with value: "keys"
+    Then Enter GCS sink property path
+#    Then Enter input plugin property: "path" with value: "gcsTargetPath"
+    Then Select GCS property format "avro"
```

Comment:
The scenario is for delimited format then why Avro is used, provide the delimited format and the required delimiter value in the field.